indico/modules/categories/models/categories.py

```diff
@@ -181,6 +181,11 @@ def __table_args__(cls):
         nullable=True,
         index=True
     )
+    event_requires_approval = db.Column(
```

Comment:
`events_require_approval` makes more sense; but based on what we discussed before: Maybe a different name would be better? Because in the end this option will allow anyone to propose a move to the category (regardless of special permissions via the ACI)...